src/fitnesse/testsystems/slim/tables/ScenarioTable.java

```diff
@@ -166,8 +166,8 @@ public String substitute(String content) throws SyntaxError {
           String arg = scenarioArgument.getKey();
           if (getInputs().contains(arg)) {
             String argument = scenarioArguments.get(arg);
-            content = StringUtils.replace(content, "@" + arg, argument);
-            content = StringUtils.replace(content, "@{" + arg + "}", argument);
+            content = StringUtils.replace(content, "@" + arg, replaceSymbols(argument));
```

Comment:
I believe it would be nicer to have only a single call to `replaceSymbols`, before the first content alteration.